plone/app/contenttypes/upgrades.py

```diff
@@ -1,262 +1,12 @@
-
-
-def use_new_view_names(context, types_to_fix=None):  # noqa
-    """Migrate old view names to new view names."""
-    # Don't reload the profile. Only change the settings.
-    logger.info('Start updating view names...')
-    portal = getSite()
-    portal_types = getToolByName(portal, 'portal_types')
-    if types_to_fix is None:
-        types_to_fix = ['Folder', 'Collection', 'Plone Site']
-    outdated_methods = [
-        'folder_listing',
-        'folder_full_view',
-        'folder_summary_view',
-        'folder_tabular_view',
-        'folder_album_view',
-        'atct_album_view',
-        'standard_view',
-        'collection_view',
-        'all_content',
-        'thumbnail_view',
-    ]
-    new_methods = [
-        'listing_view',
-        'summary_view',
-        'tabular_view',
-        'full_view',
-        'album_view',
-        'event_listing',
-    ]
-    for ctype in types_to_fix:
-        fti = queryUtility(IDexterityFTI, name=ctype)
-        if fti is None and ctype == 'Plone Site':
-            fti = portal_types.get(ctype)
-        if fti is None:
-            return
-        view_methods = [i for i in fti.getAvailableViewMethods(None)]
-        changed = False
-        for method in outdated_methods:
-            if method in view_methods:
-                view_methods.remove(method)
-                changed = True
-        for method in new_methods:
-            if method not in view_methods:
-                view_methods.append(method)
-                changed = True
-        default_view = fti.default_view
-        if default_view in outdated_methods:
-            default_view = LISTING_VIEW_MAPPING.get(default_view)
-            changed = True
-        if changed:
-            fti.manage_changeProperties(
-                view_methods=tuple(view_methods),
-                default_view=default_view,
-            )
-            logger.info('Updated view_methods for {0}'.format(ctype))
-
-    def _fixup(obj, view_map):
-        current = obj.getLayout()
-        if current in view_map:
-            default_page = obj.getDefaultPage()
-            obj.setLayout(view_map[current])
-            logger.info('Set view to {0} for {1}'.format(
-                view_map[current],
-                obj.absolute_url(),
-            ))
-            if default_page:
-                # any defaultPage is switched of by setLayout
-                # and needs to set again
-                obj.setDefaultPage(default_page)
-
-    catalog = getToolByName(portal, 'portal_catalog')
-    search = catalog.unrestrictedSearchResults
-    for portal_type in types_to_fix:
-        logger.info('Updating view names for {}'.format(portal_type))
-        brains = search(portal_type=portal_type)
-        objs = _brains2objs(brains)
-        for obj in objs:
-            _fixup(obj, LISTING_VIEW_MAPPING)
-        if portal_type == 'Plone Site':
-            _fixup(context, LISTING_VIEW_MAPPING)
-
-
-def searchabletext_collections(context):
```

Comment:
Jenkins fails in a test that tries to import `searchabletext_collections`. The test in `test_indexes` can be removed.